content/chapters/compute/lecture/slides.mdpp

```diff
@@ -0,0 +1,1105 @@
+---
+title: "OS: Compute"
+revealOptions:
+  background-color: 'aquamarine'
+  transition: 'none'
+  slideNumber: true
+  autoAnimateDuration: 0.0
+---
+
+!INCLUDE "slides/intro.md"
+
+---
+
+## 1. Processes
```

Comment:
I would add this as a single slide, similar to `## 3. Scheduling`. Also the same for the other sections.

I'm not sure if numbers / indexes are useful here. We can give them up altogether. @RazvanN7 , what do you think?

Reply:
@Alex-deVis suggested them IIRC and I went with it. I was indifferent to this, but now that we want to split our content into multiple files, it makes sense to drop the "chapter" numbers so the individual files are easier to pick up by other teachers / students. @Alex-deVis, what do you say?

Reply:
Yup, I was thinking of that split. If @Alex-deVis agrees, I say we go for that.

Reply:
Fine by me.